internal/pkg/cli/deploy.go

```diff
@@ -414,31 +431,23 @@ func (o *deployOpts) maybeDeployEnv() error {
 	return nil
 }
 
-func (o *deployOpts) loadWkld() error {
-	if err := o.loadWkldCmd(); err != nil {
-		return err
-	}
-	if err := o.deployWkld.Ask(); err != nil {
-		return fmt.Errorf("ask %s deploy: %w", o.wlType, err)
-	}
-	if err := o.deployWkld.Validate(); err != nil {
-		return fmt.Errorf("validate %s deploy: %w", o.wlType, err)
-	}
+func (o *deployOpts) loadWkld(name string) error {
```

Comment:
Is this function being a no-op temporary?

Reply:
Should have removed it entirely.